app/src/main/java/com/team41/boromi/auth/SignupFragment.java

```diff
@@ -30,33 +39,72 @@ public SignupFragment() {
    * Use this factory method to create a new instance of this fragment using the provided
    * parameters.
    *
-   * @param param1 Parameter 1.
-   * @param param2 Parameter 2.
    * @return A new instance of fragment SignupFragment.
    */
   // TODO: Rename and change types and number of parameters
-  public static SignupFragment newInstance(String param1, String param2) {
+  public static SignupFragment newInstance() {
     SignupFragment fragment = new SignupFragment();
     Bundle args = new Bundle();
-    args.putString(ARG_PARAM1, param1);
-    args.putString(ARG_PARAM2, param2);
     fragment.setArguments(args);
     return fragment;
   }
 
   @Override
   public void onCreate(Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
-    if (getArguments() != null) {
-      mParam1 = getArguments().getString(ARG_PARAM1);
-      mParam2 = getArguments().getString(ARG_PARAM2);
-    }
   }
 
   @Override
   public View onCreateView(LayoutInflater inflater, ViewGroup container,
       Bundle savedInstanceState) {
     // Inflate the layout for this fragment
-    return inflater.inflate(R.layout.fragment_signup, container, false);
+    View view = inflater.inflate(R.layout.fragment_signup, container, false);
+    activity = (MainActivity) getActivity();
+    spinner = view.findViewById(R.id.signup_loading);
+    spinner.setVisibility(View.GONE);
+    Button createAccountButton = (Button) view.findViewById(R.id.signup_signup);
+    fullNameInput = view.findViewById(R.id.signup_name);
+    emailInput = view.findViewById(R.id.signup_email);
+    userNameInput = view.findViewById(R.id.signup_username);
+    passwordInput = view.findViewById(R.id.signup_password);
+
+    createAccountButton.setOnClickListener(new OnClickListener() {
+      @Override
+      public void onClick(View view) {
+        String fullName = fullNameInput.getText().toString();
```

Comment:
fullName is unused, do you need me to update authController to add the full name as a parameter in makeSignUpRequest?
Honestly, we might not even need to store the names of the users for this app...

Reply:
Yeah we figured this was unneeded, if someone could review the user stories to see if this is a needed field then perhaps we can expand on it

Reply:
I briefly looked at it and the only thing that is mention is user "contact information", so it depends on whether we consider someone's name to be contact information

Reply:
Though if including names is considered an extra feature then it doesn't hurt to keep it